src/sidebar/components/thread-list.js

```diff
@@ -115,13 +117,12 @@ function ThreadList({ thread, $rootScope }) {
         setWindowHeight(window.innerHeight);
         setScrollPosition(scrollContainer.scrollTop);
       },
-      10,
+      50,
```

Comment:
As noted in a recent comment, rather than increasing the debounce delay, I think there is a more effective option of rounding the `scrollPosition` state variable to the nearest say, 50 or 100 pixels. Due to the fact that we render some amount of threads above and below the viewport, we can afford to make the value approximate. What this achieves in terms of rendering efficiency is that it reduces how often `ThreadList` will re-render when scrolling (only when you have scrolled _at least_ 50 or 100 pixels or whatever). A similar thing could be done with the `windowHeight` value.

It would be important to note when declaring these values if they are rounded, because that rounding limits the purposes for which they can be used.